packages/firestore/src/local/memory_query_cache.ts

```diff
@@ -134,6 +150,28 @@ export class MemoryQueryCache implements QueryCache {
     return PersistencePromise.resolve();
   }
 
+  removeTargets(
+    transaction: PersistenceTransaction,
+    upperBound: ListenSequenceNumber,
+    activeTargetIds: ActiveTargets
```

Comment:
Same comment as above, this should just be `activeTargets` to match the argument type.

Reply:
Leaving as is. It's been renamed here and on Android already.